### PR TITLE
brand landing curve (bug 1103303)

### PR DESCRIPTION
**Pull request by ngokevin**
screen shot 2015-01-12 at 5 34 05 pm

### CODE REVIEW

**Comment by ngokevin** [comment]
@spasovski

File: src/media/css/feed/brands.styl [hunk 1]
```diff
@@ -67,7 +67,27 @@
...
+    h1 {
+        desktop_mega();
+        margin: 20px 0 40px;
+        text-align: center;
+    }
+}
+
+.brand-landing-curve {
+    background: rgb(250, 250, 250);
+    border-radius: 50% 50% 0 0;
```
Comment (spasovski) [inline src/media/css/feed/brands.styl]: I always found smoother curves to happen with the nested element being masked by a hidden overflow parent but it does introduce a bunch of cruft and less vertical growth flexibility.

File: src/media/css/feed/brands.styl [hunk 2]
```diff
@@ -67,7 +67,27 @@
...
+    height: 145px;
+    position: absolute;
+    bottom: -40px;
+    width: 120%;
+    left: -10%;
```
Comment (spasovski) [inline src/media/css/feed/brands.styl]: Percentages can give messy results on mobile but this should be all desktop anyway.

**Comment by spasovski** [comment]
Looks pretty smooth locally. Nice.
r+

File: src/media/css/feed/brands.styl (hunk 1 above)
Comment (ngokevin) [inline src/media/css/feed/brands.styl]: it looks curvy to me, how much smoother would it be able to get? i don't think it's worth the extra KBs though

**Comment by ngokevin** [comment]
thanks

**Pull request merged**